include/kos.h

```diff
@@ -66,11 +66,11 @@ __BEGIN_DECLS
 #include <arch/exec.h>
 #include <arch/stack.h>
 #include <arch/byteorder.h>
+#include <arch/rtc.h>
```

Comment:
This matches the definition in the exports files. The two exported rtc functions are present in the main exports instead of the arch_dreamcast specific one.